src/main/java/de/tum/cit/aet/artemis/quiz/domain/DragItem.java

```diff
@@ -109,6 +109,10 @@ public void setInvalid(Boolean invalid) {
         this.invalid = invalid;
     }
 
+    public void setMappings(Set<DragAndDropMapping> mappings) {
```

Comment:
These setters are needed to set the mapping to null before importing an existing DragItem in the new `LearningObjectImportService`